f5router/f5router.go

```diff
@@ -62,10 +62,18 @@ type F5Router struct {
 	poolResources    map[string]*bigipResources.Pool
 }
 
+func verifyRouteURI(ru updateHTTP) error {
+	uri := ru.URI().String()
+	if strings.Count(uri, "*") > 1 {
+		return fmt.Errorf("Invalid URI: %s multiple wildcards are note supported", uri)
+	}
+	return nil
+}
+
 func makeObjectName(uri string) string {
 	var name string
-	if strings.HasPrefix(uri, "*.") {
```

Comment:
This should remain here, and the Contains call be in an `else if`.

What we're getting now is this: `*.cf.com -> _.cf.com`

We need to preserve what it used to do: `*.cf.com -> cf.com`

Reply:
The Contains statement also makes objects of name: `_vices.cf.com`.

It might look pretty odd but we should be preserving the `cf-` prefix for these object names as well, so `cf-_vices.com`, `cf-ser_.com`, and `cf-ser_vices.com`.

Reply:
Fixed

Reply:
fixed